modules/widgets/internet-defense-league.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+class Jetpack_Internet_Defense_League_Widget extends WP_Widget {
+
+	public $defaults = array();
+
+	public $variant;
+	public $variants = array();
+
+	public $campaign;
+	public $campaigns  = array();
+	public $no_current = true;
+
+	public $badge;
+	public $badges = array();
+
+	function __construct() {
+		parent::__construct(
+			'internet_defense_league_widget',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', esc_html__( 'Internet Defense League', 'jetpack' ) ),
+			array(
+				'description' => __( 'Show your support for the Internet Defense League.', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			)
+		);
+
+		// When enabling campaigns other than 'none' or empty, change $no_current to false above.
```

Comment:
I assume this was dev note in case another campaign was to be added, so someone could modify this quickly. If we remove this, we should remove the campaigns array as well since it's currently not visible.